applications/MeshingApplication/custom_processes/metrics_levelset_process.cpp

```diff
@@ -29,6 +29,13 @@ ComputeLevelSetSolMetricProcess<TDim>::ComputeLevelSetSolMetricProcess(
     Parameters default_parameters = Parameters(R"(
     {
         "minimal_size"                         : 0.1,
+        "maximal_size"                         : 1.0,
+        "sizing_parameters":
+        {
+            "reference_variable_name"          : "DISTANCE",
+            "boundary_layer_max_distance"      : 1.0,
+            "interpolation"                    : "Constant"
```

Comment:
please follow the style-guide:
```suggestion
            "interpolation"                    : "constant"
```